docs/catalog/cloud/Use energy efficient hardware.md

```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:software-engineer
+- role:cloud-engineer
+- size:medium
+---
+
+# Use energy efficient hardware
+
+## Description
+
+Cloud virtual machines are offered with different capabilties, based different hardware processors.
+
+Hence using virtual machines based on efficient processors would impact hardware efficiency and reduce carbon emissions.
+
+
+## Solution
+
+Use new cloud virtual machines that have been engineered with "cloud native processors" to efficiently run scale-out, cloud-native workloads and consume energy efficiently.
```

Comment:
Can you add a link in the references section (or provide more context here) that better describes what a "cloud native processor" is?